yt_dlp/extractor/globo.py

```diff
@@ -80,98 +90,71 @@ class GloboIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        self._request_webpage(
-            HEADRequest('https://globo-ab.globo.com/v2/selected-alternatives?experiments=player-isolated-experiment-02&skipImpressions=true'),
-            video_id, 'Getting cookies')
-
-        video = self._download_json(
-            f'http://api.globovideos.com/videos/{video_id}/playlist',
-            video_id)['videos'][0]
-        if not self.get_param('allow_unplayable_formats') and video.get('encrypted') is True:
-            self.report_drm(video_id)
-
-        title = video['title']
+        info = self._download_json(
+            'https://cloud-jarvis.globo.com/graphql', video_id,
+            query={
+                'operationName': 'getVideoView',
+                'variables': f'{{"videoId":{video_id}}}',
```

Comment:
maybe overkill but I really hate trying to decipher the `}}}`s

use `separators` param if the default whitespace is a problem for the graphql

```suggestion
                'variables': json.dumps({'videoId': video_id}),
```